pandas/core/indexes/base.py

```diff
@@ -179,8 +179,21 @@ class Index(IndexOpsMixin, PandasObject):
     str = accessor.AccessorProperty(strings.StringMethods)
 
     def __new__(cls, data=None, dtype=None, copy=False, name=None,
-                fastpath=False, tupleize_cols=True, **kwargs):
+                fastpath=False, tupleize_cols=None, **kwargs):
+        if tupleize_cols is not None:
+            warnings.warn("The 'tupleize_cols' parameter is deprecated and "
+                          "will be removed in a future version",
+                          FutureWarning, stacklevel=2)
+        else:
+            tupleize_cols = True
```

Comment:
why are you changing this?

Reply:
Because you said that we should deprecate this parameter back in #17060.  If that's no longer the case, I can just close this.